bin/GameIndex.dbf

```diff
@@ -387,7 +387,6 @@ Region = NTSC-Unk
 Serial = SCAJ-20044
 Name   = Tomb Raider - The Angel of Darkness	// aka "TRAOD"
 Region = NTSC-Unk
-//OPHFlagHack = 1 // 08.09.2014 not needed anymore and actually causes a hang
```

Comment:
Maybe it would be useful to have this comment for future reference? Doesn't hurt to have a commented out part which might potentially be useful in the near future when someone messes with the core.